src/App.tsx

```diff
@@ -1,12 +1,65 @@
 import React from 'react';
+import 'bulma';
 import './App.scss';
+import { GoodsList } from './Components/GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { get5First, getAll, getRedGoods } from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[],
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    goods: [],
+  };
+
+  LoadHandler = async (getFunction: Promise<Good[]>) => {
+    const goods = await getFunction;
+
+    this.setState({
+      goods: [...goods],
+    });
+  };
+
+  render() {
+    const { goods } = this.state;
+
+    return (
+      <section
+        className="is-half "
+      >
+        <div className="container">
+          <button
+            type="button"
+            className="button is-link"
+            onClick={() => this.LoadHandler(getAll())}
```

Comment:
don't call the function here, you should call getAll function inside LoadHandler
```suggestion
            onClick={() => this.loadHandler(getAll)}
```